docs/troubleshooting.md

```diff
@@ -139,6 +140,21 @@ kubectl delete secret pipelines-as-code-secret -n pipelines-as-code
 6. :gear: On the PR page, type `/retest` on the comment box and post the comment.
    Observe the behavior once more.
 
+# PR fails when Webhook Secret was not added
+
+When creating a github application if webhook secret was not added we get following error in our opened PR:
+
+```
+There was an issue validating the commit: "could not validate payload, check your webhook secret?: no signature has been detected, for security reason we are not allowing webhooks that has no secret"
+```
+
+
+To resolve this issue, go to the github application you created and add your webhook secret which was generated using following command
+
+```bash
```

Comment:
I am not sure about this command
I think its enough to state that we need to add the secret that we got when we created the application (or generate a new one if we do not know what it is)

Reply:
I agree. I think we should point the reader to the relevant part of the documentation rather than repeating it, and ask them to confirm that the app includes a secret.

Specifically here, this command is not a part of our documentation - it's mentioned in pipelines as code documentation, so if it changes, we'd have to change our documentation as well.